alpha/lib/model/DeliveryProfileLive.php

```diff
@@ -360,13 +358,7 @@ protected function getLivePackagerUrl($entryServerNode, $streamFormat = null)
 		$livePackagerUrl .= $serverNode->getSessionType($entryServerNode);
 
 		$entry = $this->getDynamicAttributes()->getEntry();
-		if ($entry->getExplicitLive())
-		{
-			$userType = self::USER_TYPE_ADMIN;
-		 	if (!$entry->canViewExplicitLive())
-				$userType = self::USER_TYPE_USER;
-			$livePackagerUrl .= "type/$userType/";
-		}
+		$livePackagerUrl .= $serverNode->getExplicitLiveUrl($livePackagerUrl, $entry);
```

Comment:
Access static function as instance function

Reply:
I aligned with the above string concatenations that also calling the static function that way.
From what I understand from PHP doc it is allowed 
https://www.php.net/manual/ro/language.oop5.static.php
>  A property declared as static cannot be accessed with an instantiated class object **(though a static method can).**

Do you think it is better to change it to instance function? (calling it the regular static way is impossible as we need to call the relevant implementation according to the serverNode type)